src/ado2gh/Commands/GenerateMannequinCsvCommand.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.AdoToGithub.Commands
+{
+    public class GenerateMannequinCsvCommand : Command
+    {
+        internal Func<string, string, Task> WriteToFile = async (path, contents) => await File.WriteAllTextAsync(path, contents);
+
+        private readonly OctoLogger _log;
+        private readonly GithubApiFactory _githubApiFactory;
+
+        public GenerateMannequinCsvCommand(OctoLogger log, GithubApiFactory githubApiFactory) : base("generate-mannequin-csv")
+        {
+            _log = log;
+            _githubApiFactory = githubApiFactory;
+
+            Description = "Generates a CSV with unreclained mannequins to reclaim them in bulk.";
+            IsHidden = true;
+
+            var githubOrgOption = new Option<string>("--github-org")
+            {
+                IsRequired = true,
+                Description = "Uses GH_PAT env variable."
+            };
+
+            var outputOption = new Option<string>("--output")
+            {
+                IsRequired = true,
+                Description = "Output filename"
+            };
+
+            var forceOption = new Option("--include-reclaimed")
+            {
+                IsRequired = false,
+                Description = "Map the user even if it was previously mapped"
```

Comment:
Should this say something like "Include mannequins that have already been reclaimed"